apps/web/src/components/infoscreen/kanttiinit-combined/update.ts

```diff
@@ -0,0 +1,61 @@
+import { getCurrentLocale } from "../../../locales/server";
+import type {
+  Restaurant,
+  RestaurantMenu,
+  RestaurantMenuLite,
+} from "../types/kanttiinit-types";
+import { KanttiinitMenus } from "./menus";
+import { KanttiinitRestaurants } from "./restaurants";
+
+export const fetchMenus = async (): Promise<RestaurantMenu[]> => {
+  try {
+    const locale = await getCurrentLocale();
```

Comment:
The `locale` should probably be in the React component rather than this function 🤔 Or even better in the actual page/layout/route, but that probably isn't the way the others implement this anyway so 🤷